include/deal.II/lac/solver_gmres.h

```diff
@@ -216,13 +219,27 @@ class SolverGMRES : public SolverBase<VectorType>
           LinearAlgebra::OrthogonalizationStrategy::modified_gram_schmidt);
 
     /**
-     * Maximum number of temporary vectors. This parameter controls the size
-     * of the Arnoldi basis, which for historical reasons is
-     * #max_n_tmp_vectors-2. SolverGMRES assumes that there are at least three
-     * temporary vectors, so this value must be greater than or equal to three.
+     * Maximum number of temporary vectors. Together with #max_basis_size,
+     * this parameter controls the size of the Arnoldi basis, which for
+     * historical reasons is #max_n_tmp_vectors-2. SolverGMRES assumes that
```

Comment:
The usage of `#` is the first I see it. Do we need it? `SolverGMRES` does not and still creates a link in Doxygen.

Reply:
I copied it from somewhere else in this file and then just went with it by routine; thinking about it I will remove it, thanks.